internal/bminventory/inventory_test.go

```diff
@@ -9055,6 +9055,28 @@ var _ = Describe("UpdateClusterInstallConfig", func() {
 		verifyApiError(response, http.StatusBadRequest)
 	})
 
+	It("returns bad request when validation fails - invalid json data", func() {
```

Comment:
These new tests aren't really testing anything since we're mocking the validate call. I don't think we need them.

Reply:
True, thanks for catching! Removed them and added a subsystem test instead.